test/system/appveyor-yml.test.js

```diff
@@ -15,20 +14,20 @@ describe('appveyor.yml', function () {
         appveyorYAMLError = e;
     }
 
-    it('must exist', function (done) {
+    it('should exist', function (done) {
         fs.stat('.appveyor.yml', done);
     });
 
-    it('must be a valid yml', function () {
-        expect(appveyorYAMLError && appveyorYAMLError.message || appveyorYAMLError).to.not.be.ok();
+    it('should be a valid yml', function () {
+        expect(appveyorYAMLError && appveyorYAMLError.message || appveyorYAMLError).to.be.undefined;
     });
 
     describe('strucure', function () {
-        it('init script must exist', function () {
-            expect(appveyorYAML.init[0]).to.be('git config --global core.autocrlf input');
+        it('should have init script', function () {
+            expect(appveyorYAML.init[0]).to.equal('git config --global core.autocrlf input');
```

Comment:
Assertion could be chained for better chai usage using yaml .to.have.property(‘init’).and.to.have …